src/strings/en-us.json

```diff
@@ -626,7 +627,7 @@
     "LabelEmbedAlbumArtDidl": "Embed album art in Didl",
     "LabelEmbedAlbumArtDidlHelp": "Some devices prefer this method for obtaining album art. Others may fail to play with this option enabled.",
     "LabelEnableAutomaticPortMap": "Enable automatic port mapping",
-    "LabelEnableAutomaticPortMapHelp": "Attempt to automatically map the public port to the local port via UPnP. This may not work with some router models. Changes will not apply until after a server restart.",
+    "LabelEnableAutomaticPortMapHelp": "Attempt to automatically map the public ports to local ports via UPnP. This may not work with some router models. Changes will not apply until after a server restart.",
```

Comment:
Is it the public port or the next nat layer port? Seems like a bit of a misnomer

Reply:
Any suggestion on how to word it? I'm not too familiar with all the networking nomenclature

Reply:
Hm, not sure without confusing half the users. I'm not a fan of uPnP in general. I'd like to add a warning against using it as well. the IGD Protocol is how the port forwarding is mapped with upnp. 

http://www.upnp-hacks.org/upnp.html
https://en.wikipedia.org/wiki/Internet_Gateway_Device_Protocol

Reply:
How about `Automatically forward public ports on your router to local ports via UPnP. This may not work with some router models. Changes will not apply until after a server restart.`

Assumption being the primary use case here is a simple network topology (single router direct to server, no RP...etc).  This option is probably not useful or desired in a more complex setup...

Reply:
My thoughts is how to phrase it so people using docker + bridge mode will know that it won't work.

Reply:
I added the suggestion from @PrplHaz4 with a bit of extra text explaining that this won't work for some network configurations. I'm thinking that if we want to go into additional detail on this it might make more sense to put that in the docs site (which we can link to from here if necessary), while the help text here is good enough for the most common use case. Open to other suggestions though